doc/dev/rbd-diff.rst

```diff
@@ -67,3 +67,80 @@ End
 ---
 
 - u8: 'e'
+
+
+Header
+~~~~~~
+
+"rbd diff v2\\n"
+
+Metadata records
+~~~~~~~~~~~~~~~~
+
+Every record has a one byte "tag" that identifies the record type,
+followed by length of data, and then some other data.
+
+Metadata records come in the first part of the image.  Order is not
+important, as long as all the metadata records come before the data
+records.
+
+In v2, we have the following metadata in each section:
+(1 Bytes) tag.
+(8 Bytes) length.
```

Comment:
Yes, but what about keeping 8Bytes here in case of we want to break the limit of 32M someday, and 4Bytes for each tag sounds not a big waste for an output file of a rbd export-diff.